index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/1935d064dd.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;600;700;800&family=Roboto:wght@300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portifolio</title>
+  </head>
+  <body>
+    <header>
+      <nav id="nav-bar" class="nav-bar">
+        <a href="index.html" id="logo">Filbert</a>
+        <i class="fa fa-bars" id="bars"></i>
+      </nav>
+    </header>

```

Comment:
- [ ] Kindly put the logo outside the navigation because later on when you will implement the desktop navigation and add mobile menus, it will be difficult to manage things, in the upcoming milestones of this project only the `nav` tag and its children need to show and hide on the click of hamburger button. No doubt, there is no semantic issue but logically it is not correct in the context of web development. Please look at the below suggestion to avoid the mistake.

```suggestion
    <header>
          <a href="index.html" id="logo">Filbert</a>
          <nav id="nav-bar" class="nav-bar">
              <i class="fa fa-bars" id="bars"></i>
          </nav>
    </header>

```